server/index.ts

```diff
@@ -176,16 +193,32 @@ export const configSchema = schema.object({
       // the login config here is the same as old config `_security.basicauth.login`
       // Since we are now rendering login page to browser app, so move these config to browser side.
       login: schema.object({
-        title: schema.string({ defaultValue: 'Please login to OpenSearch Dashboards' }),
+        title: schema.string({ defaultValue: 'Log in to OpenSearch Dashboards' }),
         subtitle: schema.string({
           defaultValue:
-            'If you have forgotten your username or password, please ask your system administrator',
+            'If you have forgotten your username or password, contact your system administrator.',
         }),
         showbrandimage: schema.boolean({ defaultValue: true }),
         brandimage: schema.string({ defaultValue: '' }),
         buttonstyle: schema.string({ defaultValue: '' }),
       }),
     }),
+    openid: schema.object({
+      login: schema.object({
+        buttonname: schema.string({ defaultValue: 'Log in with single sign-on' }),
+        showbrandimage: schema.boolean({ defaultValue: false }),
+        brandimage: schema.string({ defaultValue: '' }),
+        buttonstyle: schema.string({ defaultValue: '' }),
+      }),
+    }),
+    saml: schema.object({
+      login: schema.object({
+        buttonname: schema.string({ defaultValue: 'Log in with single sign-on' }),
```

Comment:
should the texts for saml and openid be the same? If so does it make sense to declare the object and then re-use it here.

Reply:
This is only the default value. User can change the button displayname on demand from config.yml.